theme/themes/pastanaga/elements/container.overrides

```diff
@@ -59,18 +59,18 @@
 
 .drag.handle.wrapper {
   @media only screen and (max-width: @largestMobileScreen) {
-    margin-left: ~'calc(50vw - (' @mobileWidth + ~'/2)) !important';
+    margin-left: calc(50vw - (@mobileWidth / 2)) !important;
```

Comment:
maybe you should keep the `~" "` around https://github.com/less/less.js/issues/974#issuecomment-9229470